tests/autorag/nodes/passagereranker/test_mixedbreadai_reranker.py

```diff
@@ -0,0 +1,80 @@
+from unittest.mock import patch
+
+import pytest
+
+import autorag
+from autorag.nodes.passagereranker import MixedbreadAIReranker
+from tests.autorag.nodes.passagereranker.test_passage_reranker_base import (
+	queries_example,
+	contents_example,
+	ids_example,
+	base_reranker_test,
+	project_dir,
+	previous_result,
+	base_reranker_node_test,
+)
+
+
+async def mock_mixedbreadai_reranker(client, query, documents, ids, top_k, **kwargs):
+	if query == queries_example[0]:
+		return (
+			[documents[1], documents[2], documents[0]][:top_k],
+			[ids[1], ids[2], ids[0]][:top_k],
+			[0.8, 0.2, 0.1][:top_k],
+		)
+	elif query == queries_example[1]:
+		return (
+			[documents[1], documents[0], documents[2]][:top_k],
+			[ids[1], ids[0], ids[2]][:top_k],
+			[0.8, 0.2, 0.1][:top_k],
+		)
+	else:
+		raise ValueError(f"Unexpected query: {query}")
+
+
+@pytest.fixture
+def mixedbreadai_reranker_instance():
+	return MixedbreadAIReranker(project_dir=project_dir, api_key="mock_api_key")
+
```

Comment:
You need to add test code for `mixedbreadai_rerank_pure` itself. Even if it will be skipped in the github action environment. 
Because you mock `mixedbreadai_rerank_pure`, you need to do it. 
And please do not afraid to mock the function inside mixedbreadai. Because I know mocking the whole instance from api is horrible and need long code, but the chatGPT can manage it.